doc/chsmack.8

```diff
@@ -26,7 +26,7 @@ chsmack \- Change or list the Smack properties of filesystem objects
 
 .B chsmack [-L|--dereference] [--] files...
 
-.B chsmack [-L|--dereference] props... [--] files...
+.B chsmack [-L|--dereference] [-D|--drop] props... [--] files...
```

Comment:
With the additional drop options that you introduce in the next patch, is there really a need for drop-all option added here? The example from the commit message could be rewritten as:
> chsmack -a User::Item -EMT

I'm in favor of the selective removal options, with this -D thing the user doesn't have to be explicit about the attributes that are supposed to be removed.